tahrir/views.py

```diff
@@ -70,6 +63,15 @@ def _get_user(request, id_or_nickname):
             return None
 
 
+def _get_team(request, team_id):
+    '''Get team for the given team_id'''
+    team = request.db.get_team(team_id=team_id)
+
+    if team:
+        return team
+    return None
```

Comment:
`get_team` returns the `team object` or `None`. I guess I can return just the team.